docs/tutorial/README.md

```diff
@@ -1,83 +1,138 @@
-
-- [Install collie](https://github.com/meshcloud/collie-cli#-install-and-usage)
-- Create a Foundation
 - Add a cloud platform to your foundation
-- Create and deploy a landing zone to this cloud platform (using an existing kit module)
+- Create and deploy a Organization Hierarchy to this cloud platform (using an existing kit module)
 
-The kit is meant to be comfortably used with [collie cli](https://github.com/meshcloud/collie-cli) using a "fork and own"
-approach - you will ulitmately build your own kit to meet your organizations unique requirements based on the public examples in this kit.
-After installing collie, start by creating a new git repository and creating a new foundation.
 
-```shell
-git init my-foundation-kit
-cd my-foundation-kit
-collie init
-collie foundation new my-foundation-dev
-```
+### 1. Start your Cloud Foundation
+
+In this and upcoming tutorials we are using the fictional Company name Likvid. 
+
+To start your cloud journey with Likvid, follow these steps:
+
+1. Create a folder for your foundation by running the command: **`mkdir -p likvid-foundation-repo && cd likvid-foundation-repo`**.
+2. Transform the folder into a Git repository using the command: **`collie init`**.
+3. Next, create the foundation itself by executing: **`collie foundation new likvid-foundation-dev`**.
+4. Add a cloud platform, in this case, Azure, by selecting `Azure` when prompted. Define an ID for the platform, such as `az` and provide a human-readable name for it, also `az`
+5. Select the account to configure the Azure Active Directory (AAD) Tenant ID and default Subscription ID. For example: `foundation-likvid-dev / AAD 00000000-1111-2222-3333-444444444444`. Save the configuration and exit the assistant.
 
-A foundation describes a number of cloud platforms (e.g. AWS and GCP), landing zones, cloud customers (development teams in your organization) and their cloud tenants (e.g. AWS Accounts and GCP projects).
-The `collie` cli will interactively
-prompt you for about the configuration settings it needs to successfully connect to your cloud platforms. After setting
-up your first foundation, your repository will look like this:
+After completing these steps, your repository structure will resemble the following:
 
 ```shell
-$ tree
+`tree
 .
-└── foundations
-    └── my-foundation-dev
-        ├── README.md
-        └── platforms
-            └── gcp
-                └── README.md
+├── [README.md](http://readme.md/)
+├── compliance
+├── foundations
+│   └── likvid-foundation-dev
+│       ├── [README.md](http://readme.md/)
+│       └── platforms
+│           └── az
+│               └── [README.md](http://readme.md/)
+└── kit
+└── [README.md](http://readme.md/)`
+
+`7 directories, 4 files`
 ```
 
-Next, let's create our first kit module and apply it to our platform.
 
-```shell
-collie kit new "my-first-gcp-module"
-collie kit apply "my-first-gcp-module" --foundation my-foundation-dev --platform gcp
+### 2. Deploying Your First Kit
```

Comment:
The order we discussed this morning in the mockup was

Tutorials 
#1 Start your Cloud Foundation
Collie init
Collie foundation new
How the repo works
Collie kit import
Collie foundation docs
Result: understand the general structure and understand the “output” = docs

#2 Deploy something
Collie kit apply
Collie foundation deploy


#3 Bootstrapping
Collie kit apply
Bootstrapping
Collie foundation deploy

Reply:
I thought the docs stuff would @felixzieger do thats the reason the sequence is not completed